src/Transfer/Resources/Database/Attributes/RelationshipAttribute.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database\Attributes;
+
+use Utopia\Transfer\Resources\Database\Attribute;
+use Utopia\Transfer\Resources\Database\Collection;
+
+class RelationshipAttribute extends Attribute
```

Comment:
```suggestion
class Relationship extends Attribute
```

No need of the Attribute suffix. Make sure to change it in the file name.